.github/workflows/dapr-test.yml

```diff
@@ -306,8 +315,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target_os: [linux, windows]
-        target_arch: [amd64]
+        os: [ubuntu-latest, windows-2019]
```

Comment:
Unlike in the "build" step, this is used in a different way and all the Actions runners are **always** on Linux/amd64. In this case, you want to keep `os` as `linux, windows` because that's not the OS of GitHub Actions, but just the OS used in AKS